2023-08-11 16:49:52,282 INFO    MainThread:68732 [internal.py:wandb_internal():90] W&B internal server running at pid: 68732, started at: 2023-08-11 16:49:52.282409
2023-08-11 16:49:52,284 DEBUG   HandlerThread:68732 [handler.py:handle_request():141] handle_request: run_start
2023-08-11 16:49:52,285 INFO    WriterThread:68732 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_164951-2vvr4135/run-2vvr4135.wandb
2023-08-11 16:49:52,286 DEBUG   HandlerThread:68732 [meta.py:__init__():35] meta init
2023-08-11 16:49:52,286 DEBUG   HandlerThread:68732 [meta.py:__init__():49] meta init done
2023-08-11 16:49:52,286 DEBUG   HandlerThread:68732 [meta.py:probe():209] probe
2023-08-11 16:49:52,287 DEBUG   HandlerThread:68732 [meta.py:_setup_git():199] setup git
2023-08-11 16:49:52,294 DEBUG   HandlerThread:68732 [meta.py:_setup_git():206] setup git done
2023-08-11 16:49:52,294 DEBUG   HandlerThread:68732 [meta.py:_save_pip():53] save pip
2023-08-11 16:49:52,295 DEBUG   HandlerThread:68732 [meta.py:_save_pip():67] save pip done
2023-08-11 16:49:52,295 DEBUG   HandlerThread:68732 [meta.py:probe():247] probe done
2023-08-11 16:49:56,693 WARNING MainThread:68732 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-11 16:49:57,003 WARNING MainThread:68732 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-11 16:49:57,003 ERROR   MainThread:68732 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-11 16:49:57,288 INFO    SenderThread:68732 [sender.py:finish():1111] shutting down sender
2023-08-11 16:49:57,694 INFO    WriterThread:68732 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_164951-2vvr4135/run-2vvr4135.wandb
2023-08-11 16:49:57,695 INFO    HandlerThread:68732 [handler.py:finish():806] shutting down handler
2023-08-11 16:49:57,696 INFO    MainThread:68732 [internal.py:handle_exit():80] Internal process exited
